diff --git a/node_modules/pokemon-showdown/.sim-dist/pokemon.js b/node_modules/pokemon-showdown/.sim-dist/pokemon.js
index 852ccf6..0e957b1 100644
--- a/node_modules/pokemon-showdown/.sim-dist/pokemon.js
+++ b/node_modules/pokemon-showdown/.sim-dist/pokemon.js
@@ -447,7 +447,7 @@ var _dex = require('./dex');
 		this.baseMaxhp = 0;
 		this.hp = 0;
 		this.clearVolatile();
-		this.hp = this.maxhp;
+		this.hp = 20; //this.hp = this.maxhp;
 	}
 
 	toJSON() {
